docs/content/dependencies.md

```diff
@@ -142,3 +142,82 @@ would all work like [the param example in the getting started guide]({{< ref
 "getting-started.md" >}}#params). We're not quite there yet, but keep an eye
 out!
 {{< /note >}}
+
+## Locking
+
+There can be some scenarios where a group of tasks are not explicitly dependent
+on each other but also cannot be run in parallel. A good example of this is
+package management tools like
+[apk](http://wiki.alpinelinux.org/wiki/Alpine_Linux_package_management) or
+[apt](https://wiki.debian.org/Apt). As an example, let's look at this file which
+installs three packages:
+
+```hcl
+task "install-tree" {
+  check = "dpkg -s tree >/dev/null 2>&1"
+  apply = "apt-get update 2>&1 > /dev/null && apt-get -y install tree"
+}
+
+task "install-jq" {
+  check = "dpkg -s jq >/dev/null 2>&1"
+  apply = "apt-get update 2>&1 > /dev/null && apt-get -y install jq"
+}
+
+task "install-build-essential" {
+  check = "dpkg -s build-essential >/dev/null 2>&1"
+  apply = "apt-get update 2>&1 > /dev/null && apt-get -y install build-essential"
+}
+```
+
+Here is what the corresponding graph looks like:
+
+{{< figure src="/images/dependencies/without-locks.png"
+           caption="The graph output of the above module. Converge will attempt
```

Comment:
probably shouldn't wrap here since this will be inserted verbatim into the alt text.